package-lock.json

```diff
@@ -753,7 +753,8 @@
     "node_modules/@brave/leo": {
       "version": "0.0.1",
       "resolved": "git+ssh://git@github.com/brave/leo.git#349a774b7cd5151db14dae6144f7e47168fe7432",
-      "integrity": "sha512-1iQ+4+ZoYlJm4RfeEUzuSoopZjzat0iIZNc0wH7wpo5iHFmcX03ptb5onu5Ck6UHB0SMR2dfOin4n9j46stM+A==",
```

Comment:
@aguscruiz you should revert this file - its not really part of your change:

```
git checkout origin/master -- package-lock.json
```

Reply:
Done!